build/azure-pipelines/win32/product-build-win32.yml

```diff
@@ -82,10 +82,23 @@ steps:
     condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'), ne(variables['NPM_REGISTRY'], 'none'))
     displayName: Setup NPM Authentication
 
+  - powershell: |
+      . build/azure-pipelines/win32/exec.ps1
+      $ErrorActionPreference = "Stop"
+      $curPath = $pwd.Path
+      cd .build
+      exec { git clone -b rzhao271/msvs-spectre https://github.com/rzhao271/node-gyp.git } "Cloning rzhao271/node-gyp failed"
+      cd node-gyp
+      npm install
+      cd "$curPath"
```

Comment:
I'm not sure which statements here should be wrapped in `exec` calls. Whenever a statement within an `exec` fails, I don't actually see why it failed.